src/main/java/baseEntities/BasePage.java

```diff
@@ -0,0 +1,15 @@
+package baseEntities;
+
+import core.ReadProperties;
+import org.openqa.selenium.WebDriver;
+
+public abstract class BasePage {
+
+    protected WebDriver driver;
+    protected ReadProperties properties;
+
+    public BasePage() {
+        driver = BaseTest.getDriver();
```

Comment:
Why do you access method of BaseTest inside BasePage ? Aren't you able to access to driver via DriverManager.getDriver() ?

Reply:
fixed